src/cancel-reminders/lib/db.ts

```diff
@@ -78,7 +78,7 @@ export async function cancelPendingSignups(
 			runWithLogging(recurringQuery, pool),
 		]).then(
 			([oneOffResult, recurringResult]) =>
-				oneOffResult.rowCount + recurringResult.rowCount,
+				(oneOffResult.rowCount ?? 0) + (recurringResult.rowCount ?? 0),
```

Comment:
`rowCount` can now be null:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/dde4bd1701da5466a0b2c0279398a7f364b79dc3/types/pg/index.d.ts#L87